docs/index.md

```diff
@@ -20,7 +20,7 @@ description: Documentation homepage for the Codacy automated code review tool.
   </div>
 
   <div class="content-link-column">
-    <div>Using Codacy</div>
+    <div>Using Codacy Quality</div>
```

Comment:
Should we also change to Codacy Quality in the description above? ("Documentation homepage for the **Codacy** automated code review tool.")

Reply:
Good point @claudiacarpinteiro, it's a good idea to change it there as well since it's the meta description for the page. :+1: